registry/locales/cli.yml

```diff
@@ -0,0 +1,87 @@
+_version: 2
```

Comment:
I believe Windows resources are typically one folder/file per locale, so we may want to stick with version 1

Reply:
I'm not strictly opposed to that convention, especially if it's more familiar to contributors—I went with v2 because of how I was organizing the translations and believe that they'll be easier to maintain compared to replicating the structure across each file.

Reply:
The other thing to consider is that I've been using `toml` and organizing it by source files and keeping it in order with the source file to make it a bit easier to map later